scripts/gh/seed-overview-depths.sh

```
@@ -0,0 +1,77 @@
+  local num="$1" depth="$2"
+  local item
+  if ! item=$(add_or_get_item "$num"); then
+    echo "Skip #$num (cannot obtain project item)" >&2
+    return 1
+  fi
+  if [[ -z "$item" || "$item" == null ]]; then
+    echo "Skip #$num (empty item id)" >&2
+    return 1
+  fi
+  if gh api graphql \
+      -f query='mutation($proj:ID!,$item:ID!,$field:ID!,$num:Float!){ updateProjectV2ItemFieldValue(input:{projectId:$proj, itemId:$item, fieldId:$field, value:{number:$num}}){ projectV2Item{ id } } }' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)